### PR TITLE
Unifies Range Slider in homepage and game edition

### PR DESCRIPTION
**Pull request by romgar**
### Slider

Before:

Screen Shot 2021-04-24 at 16 12 05

After:

Screen Shot 2021-04-25 at 13 07 46

### Slider Range

Before:

Screen Shot 2021-04-24 at 14 17 47

After:

Screen Shot 2021-04-24 at 15 33 47

Related to #319

### CODE REVIEW

File: src/components/boardComponents/ItemFormFactory.jsx [hunk 1]
```diff
@@ -104,13 +104,13 @@ const ItemFormFactory = () => {
                     step={5}
                     included={false}
                     marks={{
+                      "-180": -180,
                       "-90": -90,
                       "-45": -45,
-                      "-30": -30,
```
Comment (jrmi) [inline src/components/boardComponents/ItemFormFactory.jsx:109]: The 30° value is convenient for hexagons ;-)

File: src/views/GameListView.jsx [hunk 2]
```diff
@@ -302,7 +304,9 @@ const GameListView = () => {
             <li className="duration-filter">
               <span className="filter-title">{t("Duration (mins)")}</span>
               <SliderRange
-                defaultValue={[15, 90]}
+                defaultValue={[2, 4]}
```
Comment (jrmi) [inline src/views/GameListView.jsx:307]: Bad default values?

**Review by jrmi — CHANGES_REQUESTED** [review]

**Comment by jrmi** [comment]
image

Can you add little gap to the right like other inputs?

File: src/components/boardComponents/ItemFormFactory.jsx (hunk 1 above)
Comment (romgar) [reply src/components/boardComponents/ItemFormFactory.jsx:109]: Got it, I need to change the font size then.

File: src/views/GameListView.jsx (hunk 2 above)
Comment (romgar) [reply src/views/GameListView.jsx:307]: Seems like a bad copy/paste to me. Fixing.

**Review by jrmi — APPROVED** [review]

**Pull request merged**